09_Time_Series/Investor_Flow_of_Funds_US/Exercises_with_code_and_solutions.ipynb

```diff
@@ -1055,8 +1051,257 @@
     }
    ],
    "source": [
-    "monthly = monthly.dropna()\n",
-    "monthly"
+    "monthly = df.resample('M').sum()\n",
```

Comment:
dropna() was not working, because 0 is not NaN.
I used drop_duplicates() (seems quite a hacky way) and simple brute force method (check all columns are non-zero).